### DISCUSSION TITLE
Footnotes in tables

### DISCUSSION THREAD

**Discussion by myrthos** [gethinode/hinode#461] [Help] [answered]
Markdown footnotes in tables work, which is good, but they do not work when the `table` shortcode is used. In that case they are just rendered as plain text.

Footnotes do also work when not using the shortcode but the `{.table}` formatting option. However in that case a very wide table is rendered into the right hand bar and even off the screen. I understand I should use the shortcode for that, but it doesn't support footnotes. And I just happen to have that combination of a wide table and a footnote.

Is it intentional that the `table` shortcode does not support footnotes?

**Comment by markdumay** [comment, marked as answer]
It's seemingly a [limitation of Hugo](https://discourse.gohugo.io/t/isolation-of-links-and-footnotes-inside-codeblocks/43795/7). Unfortunately, no render hook is available to address this.

What you could consider is to include the footnote(s) within the table shortcode itself. The footnote will be rendered directly below the table instead of the bottom of the page though. It also does not respect (the numbering of) any other footnotes on the same page.

```
{{< table "table-striped" >}}
| #  | Item        | Left aligned | Center aligned |   Right aligned|
| -- | ----------- |:-------------|:--------------:| --------------:|
| 1. | First item  | some text[^2]| more text      | even more text |
| 2. | Second item | some text    | more text      | even more text |
| 3. | Third item  | some text    | more text      | even more text |
[^2]: Shortcode footnote.
{{< /table >}}
```

**Comment by myrthos** [comment]
I think I can work with that. Thanks for checking it.
